lua/lualine.lua

```diff
@@ -149,7 +146,6 @@ local function component_loader(component)
         component.color, component.component_name, component)
       end
       update_color()
-      utils.expand_set_theme(update_color)
```

Comment:
You shouldn't remove expand_set_theme from anywhere . They are there to load the colors on ColorScheme event . So if users change colorscheme the highlights remain . As all colorscheme at start remove all set highlights. It's unrelated to lualine themes . Previously we were doing a separate autocmd for each time we set highlights . That's why I had added that function to simplify cutom highlights so we can do it with just one autocmd (That is added in set_autocomand func(or something like that).